java/java.lsp.server/vscode/BUILD.md

```diff
@@ -0,0 +1,98 @@
+# VSNetBeans: A NetBeans based Extension for VSCode
```

Comment:
IMHO title should correspond to the description in package.json: Apache NetBeans Language Server Extension for VS Code